controllers/csiaddons/reclaimspacecronjob_controller_test.go

```diff
@@ -108,3 +109,86 @@ func TestGetScheduledTimeForRSJob(t *testing.T) {
 		})
 	}
 }
+
+func TestGetNextSchedule(t *testing.T) {
+	type args struct {
+		rsCronJob *csiaddonsv1alpha1.ReclaimSpaceCronJob
+		now       time.Time
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    time.Time
+		want1   time.Time
```

Comment:
I always prefer understandable names, `want` is always er to `time.Time{}` which makes it a little useless?

`want1` might be renamed to `nextTime` or something that expresses its meaning a little more?

Reply:
Done.